documentation/getting-started/rollbacks/index.markdown

```diff
@@ -3,4 +3,18 @@ title: Rollbacks
 layout: default
 ---
 
-**Missing Content**
+When a deployment is run, Capistrano executes one task at a time on all servers and waits for that task to be done before moving on to the next one. If a task fails on one or more servers, the errors are printed, Capistrano waits for all servers to either return an error or succeed, and then stops execution of any further tasks.
```

Comment:
This is not necessarily true. Generally speaking, Capistrano runs a task on all servers in parallel. It then waits for the first server to complete (or fail), then the second, and so on. If a failure is detected, Capistrano exits without waiting for the remaining servers to finish.

For example, if you have servers 1, 2, and 3, and 3 fails, Capistrano won't "know" until 1 and 2 are done. In this case all servers execute before Capistrano exits.

If 1 fails, Capistrano exits right away before checking the status of 2 and 3. If 2 and 3 haven't finished yet, the SSH connection is simply terminated (AFAIK).

Not sure how to summarize that simply in the doc, but there you go. :)